src/util/completeFromAka.js

```diff
@@ -141,9 +152,6 @@ module.exports = (obj, akaData, dataSource, sendLog) => {
                     sendLog(logLevel.error, logDetails.error.ERR_DATA_SOURCE);
             }
         }
-        else {
```

Comment:
send log in wrapping function of data compilation

Reply:
by log you mean the warning that it wasnt completed by either aka or city?, if so then done